<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Numpad</class>
 <widget class="QWidget" name="Numpad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="NumpadButtons" name="btn7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="NumpadButtons" name="btn4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="NumpadButtons" name="btn1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="NumpadButtons" name="btn8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="NumpadButtons" name="btn5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="NumpadButtons" name="btn2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="NumpadButtons" name="btn9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="NumpadButtons" name="btn6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="NumpadButtons" name="btn3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPeruuta">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
  </widget>
  <widget class="NumpadButtons" name="btn0">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNostaSumma">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Nosta summa</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>511</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumpadButtons</class>
   <extends>QPushButton</extends>
   <header location="global">Numpadbuttons.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
